max/kernels/src/layout/tensor_core_async.mojo

```diff
@@ -263,25 +263,25 @@ fn select_k_atom[
     matrix structure that can be efficiently processed by tensor cores.
 
     Parameters:
-        type: Element data type of the tensor.
+        dtype: Element data type of the tensor.
         swizzle_mode: Memory access pattern swizzling mode.
 
     Returns:
         `Layout` - A core matrix layout optimized for tensor core operations.
     """
     alias a = _select_k_atom_bits[swizzle_mode]()
-    return upcast(a, size_of[type]() * 8)
+    return upcast(a, dtype.bitwidth())
```

Comment:
Ideally with the PR -> since it's goal is it simple rename `type` -> `dtype`, non-naming changes like this should probably be left to a separate PR since this change isn't noted in the PR change log.
There a few other instances of this too in the PR